components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/impl/DASSessionDataPublisherImpl.java

```diff
@@ -108,11 +120,21 @@ public void doPublishSessionUpdate(SessionData sessionData) {
         publishSessionData(sessionData, AuthPublisherConstants.SESSION_UPDATE_STATUS);
     }
 
+    @Override
+    public void init(InitConfig initConfig) {
+
+    }
+
     @Override
     public String getName() {
         return AuthPublisherConstants.DAS_SESSION_PUBLISHER_NAME;
     }
 
+    @Override
+    public int getPriority(MessageContext messageContext) {
```

Comment:
Any reason to implement this method ? Why do we have to use a hard coded value 0 ?